modules/persistent-storage-csi-efs-sts.adoc

```diff
@@ -0,0 +1,101 @@
+
+Perform this procedure after installing the AWS EFS CSI Operator, but before installing the AWS EFS CSI driver as part of _Installing the AWS EFS CSI Driver Operator_ procedure. If you perform this procedure after installing the driver and creating volumes, your volumes will fail to mount into pods.
+
+.Prerequisites
+
+* AWS account credentials
+
+.Procedure
+
+To configure the AWS EFS CSI Driver Operator with STS:
+
+. Extract the CCO utility (`ccoctl`) binary from the {product-title} release image, which you used to install the cluster with STS. For more information, see "Configuring the Cloud Credential Operator utility".
+
+. Create and save an EFS `CredentialsRequest` YAML file, such as shown in the following example, and then place it in the `credrequests` directory:
++
+.Example
++
+[source, yaml]
+----
+apiVersion: cloudcredential.openshift.io/v1
+kind: CredentialsRequest
+metadata:
+  name: openshift-aws-efs-csi-driver
+  namespace: openshift-cloud-credential-operator
+spec:
+  providerSpec:
+    apiVersion: cloudcredential.openshift.io/v1
+    kind: AWSProviderSpec
+    statementEntries:
+    - action:
+      - elasticfilesystem:*
+      effect: Allow
+      resource: '*'
+  secretRef:
+    name: aws-efs-cloud-credentials
+    namespace: openshift-cluster-csi-drivers
+  serviceAccountNames:
+  - aws-efs-csi-driver-operator
+  - aws-efs-csi-driver-controller-sa
+----
+
+. Run the `ccoctl` tool to generate a new IAM role in AWS, and create a YAML file for it in the local file system (`<path_to_ccoctl_output_dir>/manifests/openshift-cluster-csi-drivers-aws-efs-cloud-credentials-credentials.yaml`).
++
+[source, terminal]
+----
+$ ccoctl aws create-iam-roles --name=<name> --region=<aws_region> --credentials-requests-dir=<path_to_directory_with_list_of_credentials_requests>/credrequests --identity-provider-arn=arn:aws:iam::<aws_account_id>:oidc-provider/<name>-oidc.s3.<aws_region>.amazonaws.com
+----
++
+* `name=<name>` is the name used to tag any cloud resources that are created for tracking.
+
+* `region=<aws_region>` is the AWS region where cloud resources are created.
+
+* `dir=<path_to_directory_with_list_of_credentials_requests>/credrequests` is the directory containing the EFS CredentialsRequest file in previous step.
+
+* `<aws_account_id>`` is the AWS account ID.
++
+.Example
++
+[source, terminal]
+----
+$ ccoctl aws create-iam-roles --name my-aws-efs --credentials-requests-dir credrequests --identity-provider-arn arn:aws:iam::123456789012:oidc-provider/my-aws-efs-oidc.s3.us-east-2.amazonaws.com
+----
++
+.Example output
++
+[source, terminal]
+----
+2022/03/21 06:24:44 Role arn:aws:iam::123456789012:role/my-aws-efs -openshift-cluster-csi-drivers-aws-efs-cloud- created
+2022/03/21 06:24:44 Saved credentials configuration to: credrequests/manifests/openshift-cluster-csi-drivers-aws-efs-cloud-credentials-credentials.yaml
```

Comment:
Suggest to remove "credrequests" to aviod misunderstanding.
we usually stay in `pwd` dir, under this, "credrequests" is the input dir and "manifests" is output dir, no change to generate credentials configuration in "credrequests" dir with the example cmd.